test/controllers/api/v1/code_schools_controller_test.rb

```diff
@@ -20,7 +20,7 @@ class Api::V1::CodeSchoolsControllerTest < ActionDispatch::IntegrationTest
 
   test ":index endpoint returns a JSON list of all CodeSchools" do
     get api_v1_code_schools_path, as: :json
-    assert_equal JSON.parse(response.body)[0]["name"], "Wyncode Academy"
+    assert_equal "CoderSchool", JSON.parse(response.body)[0]["name"]
```

Comment:
Perhaps we should add in another assertion around the `location` JSON data being present, as well.

Reply:
@hpjaj I added an `assert_not_nil` because we expect all `CodeSchools` to have a location. 
![screen shot 2017-10-13 at 8 37 54 am](https://user-images.githubusercontent.com/16382165/31526646-61167eba-aff2-11e7-8a6f-0b3a235b35be.jpg)

I also just checked the show action and I think I've done it the requested way. The show action includes a single `CodeSchool` as well as a location. However, I'm confused because the issue says "but with just one location." This strikes me as odd because how would we know which location to show to the user? Please advise so I can move forward with this.
![screen shot 2017-10-13 at 8 41 06 am](https://user-images.githubusercontent.com/16382165/31526699-ac260f42-aff2-11e7-827b-e6f33e96272d.jpg)

Reply:
@PrimeTimeTran - Good point.

@hollomancer @kylemh @jjhampton @Cooperbuilt - What payload do you want the `:show` endpoint to deliver?

1 - a single school at **one** location, or 
2 - a single school, with all its locations

For option 1 we would have the frontend pass two params in the request, a school and a location.
For option 2, it would just pass the school.